Homework_8/Homework8.html

```diff
@@ -0,0 +1,180 @@
+    <button id="firstBtn">Add to Shopping Cart </button>
+    </li>
+<li><h5>${dressList[1].name}</h5>
+    <img  class ="dressImage" src="${dressList[1].imageUrl}" />
+    <p>${dressList[1].description}</hp>
+    <p>Price:${dressList[1].Price}</p>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <p>Delivery Cost :${dressList[1].shippingCost.cost.deliveryCharge}</p>
+    <button id="secondBtn">Add to Shopping Cart </button>
+    </li>
+    <li><h5>${dressList[2].name}</h5>
+    <img  class ="dressImage" src="${dressList[2].imageUrl}" />
+    <p>${dressList[2].description}</p>
+    <p>Price:${dressList[2].Price}</p>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <p>Delivery Cost :${dressList[2].shippingCost.cost.deliveryCharge}</p>
+    <button id="thirdBtn">Add to Shopping Cart </button>
+    </li>
+    
+    <li><h5>${dressList[3].name}</h5>
+    <img  class ="dressImage" src="${dressList[3].imageUrl}" />
+    <p>${dressList[3].description}</p>
+    <p>Price:${dressList[3].Price}</p>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <p>Delivery Cost :${dressList[3].shippingCost.cost.deliveryCharge}</p>
+    <button id="fourthBtn">Add to Shopping Cart </button>
+    </li>` ;
+
+    let showCart = [] ;
+
+    function addFirstItem (){
+showCart.push({
+    dressName : dressList[0].name,
+    dressPrice : dressList[0].Price
+})
+    };
+    function addSecondItem (){
+showCart.push({
+    dressName : dressList[1].name,
+    dressPrice : dressList[1].Price
+})
+    };
+    function addThirdItem (){
+showCart.push({
+    dressName : dressList[2].name,
+    dressPrice : dressList[2].Price
+})
+    };
+    function addFourthItem (){
+showCart.push({
+    dressName : dressList[3].name,
+    dressPrice : dressList[3].Price
+})
+    };
+    document.getElementById("firstBtn").addEventListener("click",addFirstItem);
+    document.getElementById("secondBtn").addEventListener("click",addSecondItem);
+    document.getElementById("thirdBtn").addEventListener("click",addThirdItem);
+    document.getElementById("fourthBtn").addEventListener("click",addFourthItem);
+
+    function addShoppingCartList (){
+        let showListCmp = document.getElementById("showList");
+        for(let i=0 ; i<showCart.length ; i++){
+let newShowList = document.createElement("div")
+newShowList.innerHTML = `<span>${showCart[i].dressName}</span> <span>${showCart[i].dressPrice}</span>`;
+showListCmp.appendChild(newShowList);
```

Comment:
it would be nice if you add a more styles for displaying your shopping cart, that way you can practice more css.